### PR TITLE
Aws ts wordpress fargate rds

### PR DESCRIPTION
**Pull request by MitchellGerdisch**
This is a Typescript version of the aws-py-wordpress-fargate-rds example.
https://github.com/pulumi/examples/tree/master/aws-py-wordpress-fargate-rds

### CODE REVIEW

File: aws-ts-wordpress-fargate-rds/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+   ```
+
+1. Set the AWS region:
+
+   ```bash
+   $ pulumi config set aws:region us-west-2
+   ```
+
+1. Run `pulumi up` to preview and deploy changes. After the preview is shown you will be
+   prompted if you want to continue or not. Note: If you set the `db_password` in the configuration as described above, you will not see the `RandomPassword` resource below.
```
Comment (phillipedwards) [inline aws-ts-wordpress-fargate-rds/README.md:43]: update to match config key

File: aws-ts-wordpress-fargate-rds/network.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    const cidrBlock = args.cidrBlock || "10.100.0.0/16";
+    const instanceTenancy = args.instanceTenancy || "default";
+    const enableDnsHostnames = args.enableDnsHostnames || true;
+    const enableDnsSupport = args.enableDnsSupport || true;
+
+    const vpc = new ec2.Vpc(vpcName, {
+      cidrBlock: cidrBlock,
+      instanceTenancy: instanceTenancy,
+      enableDnsHostnames: enableDnsHostnames,
+      enableDnsSupport: enableDnsHostnames,
```
Comment (phillipedwards) [inline aws-ts-wordpress-fargate-rds/network.ts:37]: update to appropriate variable `enabledDnsSupport`

File: aws-ts-wordpress-fargate-rds/network.ts [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+      const vpcSubnet = new ec2.Subnet(subnetName, {
+        assignIpv6AddressOnCreation: false,
+        vpcId: vpc.id,
+        mapPublicIpOnLaunch: true,
+        cidrBlock: `10.100.${subnets.length}.0/24`,
+        availabilityZoneId: az,
+        tags: {"Name": subnetName},
+      }, { parent: this });
+
+      const rtAssociation = new ec2.RouteTableAssociation(`vpc-route-table-assoc-${i}`, {
```
Comment (phillipedwards) [inline aws-ts-wordpress-fargate-rds/network.ts:71]: remove unused variable

**Review by phillipedwards — CHANGES_REQUESTED** [review]

File: aws-ts-wordpress-fargate-rds/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## Getting Started
+
+There are no required configuration parameters for this project since the code will use defaults or generate values as needed - see the beginning of `index.ts` to see the defaults.
+However, you can override these defaults by using `pulumi config` to set the following values (e.g. `pulumi config set service_name my-wp-demo`).
+
+- `service_name` - This is used as a prefix for resources created by the Pulumi program.
+- `db_name` - The name of the MySQL DB created in RDS.
+- `db_user` - The user created with access to the MySQL DB.
+- `db_password` - The password for the DB user. Be sure to use `--secret` if creating this config value (e.g. `pulumi config set db_password --secret`).
```
[the remark this answers is not in the document]
Comment (MitchellGerdisch) [reply aws-ts-wordpress-fargate-rds/README.md]: fixed

File: aws-ts-wordpress-fargate-rds/README.md (hunk 1 above)
Comment (MitchellGerdisch) [reply aws-ts-wordpress-fargate-rds/README.md]: Fixed

File: aws-ts-wordpress-fargate-rds/network.ts (hunk 2 above)
Comment (MitchellGerdisch) [reply aws-ts-wordpress-fargate-rds/network.ts]: fixed

File: aws-ts-wordpress-fargate-rds/network.ts (hunk 3 above)
Comment (MitchellGerdisch) [reply aws-ts-wordpress-fargate-rds/network.ts]: fixed

**Review by phillipedwards — APPROVED** [review]

**Comment by phillipedwards** [comment]
LGTM

**Pull request merged**